docs/checker.go.tpl

```diff
@@ -0,0 +1,23 @@
+// +build ignore
```

Comment:
1. I think this line can be removed.

2. `tpl` extension is too limited.  IMO we would have fully expanded `.template`

Reply:
We cannot remove build ignore, it is needed to avoid conflicting with other `packages` in current directory .tpl is acceptable name most projects like hugo etc support this. We don't need to call it .template